datasets/cdt/README.md

```diff
@@ -0,0 +1,125 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/
```

Comment:
can you fill the tags using the tagger please ?

Reply:
done